### DISCUSSION TITLE
Template / Sample

### DISCUSSION THREAD

**Discussion by MX5Marc** [jason0611/FS22_DashboardLive#69] [Q&A]
Hi,

I've been trying to setup DBL in my tractor without much success.  Is there a template mod that I can use as a reference so I can see how the i3d and xml have been done so I can copy to my own tractor?  Or a link to a tractor that has a complete DBL that I can use as a reference?

Thank you

**Comment by jason0611** [comment]
Hi, please use the Xerion as a reference: https://github.com/jason0611/FS22_Xerion_DBL. :-)
